src/Blorc.OpenIdConnect/Models/OidcProviderOptions.cs

```diff
@@ -52,5 +53,14 @@ public OidcProviderOptions()
 
         [JsonPropertyName("accessTokenExpiringNotificationTimeInSeconds")]
         public int AccessTokenExpiringNotificationTimeInSeconds { get; set; }
+
+        [JsonPropertyName("resource")]
+        public string? Resource { get; set; }
```

Comment:
Is `Resource` sufficient or should we add `Audience` as well?

Reply:
`Audience` is a distinct parameter.
 
![image](https://github.com/WildGums/Blorc.OpenIdConnect/assets/1785664/f0225a9f-d915-4c79-8f66-a635fde13a4f)

I didn’t find any reference to the `Resource` parameter in the Keycloak documentation. I’m not sure yet, but it seems like it’s being ignored by Keycloak.

In Keycloak resources are enabling [policy enforcement] https://www.keycloak.org/docs/latest/authorization_services/#_enforcer_overview)

![image](https://github.com/WildGums/Blorc.OpenIdConnect/assets/1785664/18b3bbc9-b369-432d-a372-2ddf245c3f77)

Reply:
We use `Audience` in server-side configuration, not as part of client-side configuration.

    options.Audience = "demo-api";

So, this is an extension for other options already implemented in oidc-client-ts. So, it could work for us.

Reply:
BTW, we use `Scope` to specify the `audience` 

![image](https://github.com/WildGums/Blorc.OpenIdConnect/assets/1785664/973253b2-f4d4-43d7-a92f-cd2b38e86a3b)

Reply:
Thanks for the great research!

Reply:
`resource` is also the standard parameter for [Resource Indicators](https://datatracker.ietf.org/doc/html/rfc8707).